style/workLoggingSystem.css

```diff
@@ -131,6 +159,13 @@ button:hover {
 .part {
     padding: 1% 1% 1% 1%;
 }
+.logBtn{
+    width: 30%;
+}
```

Comment:
Please avoid introducing new class, try to use id to changes css

Reply:
this id is used at two-place, so I have to change the id.

Reply:
You can have 2 different id, but try to do it in one id.